config/v1/types_build.go

```diff
@@ -15,21 +15,21 @@ type Build struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 	// Spec holds user-settable values for the build controller configuration
 	// +optional
-	Spec BuildSpec `json:"spec,omitempty"`
```

Comment:
@adambkaplan these changes here in `types_build.go` do not change serialization behaviour. But please double check you really meant what you specified with tags (and fix in a follow-up if you haven't).

Reply:
@sttts looks good to me